Synview-backend/websocket/websocket_server.ts

```diff
@@ -7,6 +7,15 @@ import { createLogger, LogLevel } from "../../common/Logger.ts";
 import type { UserData } from "../../common/types.ts";
 
 const logger = createLogger("Backend [WS]", LogLevel.ERROR);
+const bc = new BroadcastChannel("across-server-updates");
```

Comment:
I don't think this is right. This basically gets rid of the point of having more than 1 server, because now all servers send all updates to all other servers. 

It should be more specific here - channels like this should be user_id based, and so only servers who that user is actively connected to should get the broadcast